pkg/explorer/elements/transaction.go

```diff
@@ -29,7 +29,7 @@ func (e *elements) IsTransactionConfirmed(txid string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	return data["confirmed"].(bool), nil
+	return data["confirmations"].(float64) > 0, nil
```

Comment:
usually better always checking the second return value `ok` when casting an `interface{}`

```
nConf, ok := data["confirmations"].(float64)
if !ok {
....
}
```

Reply:
the problem was on the field, `confirmations` always exists in the response